shm/Map.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include <vector>
+#include "island.hpp"
+#include <random>
+#include <algorithm>
+#include <utility>
+#include <iostream>
+#include "coordinates.hpp"
+
+constexpr size_t COORDINATE_MIN = 0;
+constexpr size_t COORDINATE_MAX = 99;
+constexpr size_t ISLANDS_COUNT = 10;
+
+class Map
```

Comment:
Please document class in header. You should write description of class and interface. Sometimes when private functions are compilated you can also document it.

```C++
// Class responsible for manage Currency in game. 
class Money {
public:
    // Function will rise price for every item in game.
    void makeInflation();

private:
}|;
```